backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -206,3 +207,14 @@ $z-index-navbar-flyout:          1000 !default;
 // Sidebar
 //--------------------------------------------------------------
 $border-sidebar:                 1px solid $color-sidebar-border !default;
+
+// Media queries
+//--------------------------------------------------------------
+$desktop-width: 959px !default;
+$tablet-width: 768px !default;
+$mobile-width: 480px !default;
+
+$large-device: "only screen and (min-width: #{$tablet-width})";
+$desktop: "only screen and (max-width: #{$desktop-width})";
+$tablet: "only screen and (max-width: #{$tablet-width})";
+$mobile: "only screen and (max-width: #{$mobile-width})";
```

Comment:
sorry, I don't get it. This way it will use the bootstrap sizes for `xl` etc screen sizes but I think we need to use the skeleton ones which are defined [here](https://github.com/solidusio/solidus/blob/master/frontend/app/assets/stylesheets/spree/frontend/_skeleton.scss) that could not match, right?